lib/miq_tools_services/bugzilla.rb

```diff
@@ -26,7 +40,7 @@ def service
     def self.ids_in_git_commit_message(message)
       ids = []
       message.each_line.collect do |line|
-        match = %r{^(Fixes|Resolves)?\s*https://bugzilla\.redhat\.com//?show_bug\.cgi\?id=(?<bug_id>\d+)$}.match(line)
+        match = /^(#{CLOSING_KEYWORDS.join("|")})?\s*#{URL_REGEX}$/i.match(line)
```

Comment:
Maybe also allow colon (":") here.  People are sometimes gonna type:

> Fixes: &lt;url&gt;

We should handle both cases (with and without).